hw/ip/flash_ctrl/data/flash_ctrl.hjson

```diff
@@ -1685,6 +1688,8 @@
           { bits: "0",
             name: "init_wip",
             desc: "Flash phy controller initializing"
+            tags: [ // Bit changes immediately after start from reset value to 1b1 due to initialization
```

Comment:
Should we start running all flash ctrl tests only after this bit de-asserts? @tjaychen, WDYT?

Reply:
> yes i think so. We should probably place this under dut_init right? is that what you mean?

Yes, exactly. That way this exclusion won't be needed.

Reply:
I am not sure I fully understand the need for this exclusion though - all flash ctrl CSR tests are passing. What test does this fix?

Reply:
> I am not sure I fully understand the need for this exclusion though - all flash ctrl CSR tests are passing. What test does this fix?

Test flash_ctrl_csr_mem_rw_with_rand_reset is currently failing and this is FIX to that. 
https://reports.opentitan.org/hw/ip/flash_ctrl/dv/latest/results.html

Reply:
> Should we start running all flash ctrl tests only after this bit de-asserts? @tjaychen, WDYT?

There are two bits, one bit for Flash Controller FLASH_CTRL.STATUS @ 0x150 and the other in FLASH_CTRL.PHY_STATUS @ 0x178. I recommend that we wait for both bits to de-assert at beginning of each test (before actual sequence is started).

Reply:
yeah that sounds good to me.
I guess the exception will be the test cases where you want to explicitly test the init bits (in contention with other traffic) right?

Reply:
Both bits will be polled at begging on each test in order to have Flash Ctrl initialized before sending transactions. Only functionality described in spec is "During the initialization process, the flash controller reads out the root seeds before allowing other usage of the flash controller". If something more needs to be tested, please state.

Reply:
yes that statement is true, but there's a bit of a subtle point here.
When the initialization is called, other activity "should" be pushed off.

But that doesn't mean software cannot issue some reads BEFORE initialization is called. So it would be good for most test cases to always poll on the init complete before doing anything else.  However, we should also have a test case where we do some reads / program and THEN initialize.  This might be useful long term as a software work-around.

Reply:
Polling will be put in task which will have knob so we can control when to poll.

Reply:
yes that sounds good. I actually recall in your original testplan you already planned to have cases where you send traffic before initialization, am i right?

The reality is that initialization is a bad name here...it's really just enabling the hardware FSM.  So software access both before and after should be allowed.

Reply:
currently in smoke test it is being wait that [csr_utils::csr_spinwait] ptr flash_ctrl_core_reg_block.status.init_wip == 0x0.
In spec it is written "It is recommended that instead of blindly issuing transactions to the flash controller, the software polls STATUS.INIT_WIP until it is 0."
We can test desired behaviour in robustness test (if we don't wait for this bit).

Reply:
Wait, I thought we decided to do the `csr_spinwait` to get the failing mem test passing, but looks like this PR is already merged?

Reply:
Sorry, I thought we had a conclusion on this and merged the PR. @nikola-miladinovic please create another PR if some changes are needed for this.